messaging/src/main/java/org/axonframework/eventhandling/tokenstore/jpa/JpaTokenStore.java

```diff
@@ -206,6 +205,7 @@ protected TokenEntry loadOrCreateToken(String processorName, int segment, Entity
      */
     public static class Builder {
 
+        private LockModeType loadingLockMode = LockModeType.PESSIMISTIC_WRITE;
```

Comment:
It would be good to add this default to the `Builder` class level javadoc and the `builder()` function javadoc as well, along side the other default descriptions in those spaces.

Reply:
I can agree to that, although I do consider this an "advanced option" that people will hardly ever need to tune anyway. Adding it to javadoc may trigger more tuning than would be healthy ;-)

Reply:
So, you'd prefer to omit that description?

Reply:
Alright, approved.